### ISSUE TITLE
Forum display icon errors

### ISSUE THREAD

**Issue by tgtje** [e107inc/e107#1085]
Using files ( latest about 20 hours) forum viewforum 

using bootstrap 3  theme all icons (new/unread etc.. ) are displayed
using bootstrap all icons (old types) are NOT displayed 

using the copy url for NON displayed image it is copied as :
 h *ttp://DOMAIN/SUBDOMAIN/forum/e107_plugins/forum/images/icons/new_small.png

notice forum is BEFORE e107_plugins..

 run an element inspect is the url displayed as <**** img src=" e107_***plugins/forum/images/icons/new_small.png" alt="Unread post exists" title="Unread post exists"> which is correct when using that one..(paste in browse-bar)

![ball](https://cloud.githubusercontent.com/assets/1811857/8386887/329f78ae-1c54-11e5-940f-98eb8203a210.jpg)

**Comment by mcpeace-maw** [comment]
Bootstrap 3 theme have in theme.php at the end of the file all image icons replaced by font icons, that is the reason you have icons in forum using that theme, it use font icons. Bootstrap theme use forum_icons_template file from forum plugin and the image icons and have this issue.

**Comment by tgtje** [comment]
Thank you for answering, but it is not a personal problem, all files are new, so it is a setting that prohibits the displaying of the 'old type' style. 
It should display fine ( always has untill i now discovered) so there is a change of influence..
Not all users will use Bootstrap 3

**Comment by tgtje** [comment]
In regards to latest updates of files (it works, however using 'live' a subdomain or on pc xampp)
some ( 3 ) icons will refuse to show up (no new posts; sticky; No new posts [popular])  

Using the files for testing the pull request from mcpeace-maw  (not merged into e107 at this point)
The outcome is that due to css styling is better !! no new posts has an icon !! but the other two refuse to display
Furthermore: when logged in i loose the admin area icon ( cogs) on BS  (not on BS3)
Notice for me on BS3 the No new posts [popular]  is now also without icon  ( = both on BS and BS3).

Just to inform

**Comment by mcpeace-maw** [comment]
Don't use latest pull for forum_icons_template.php file (will not be merged), use latest forum_icons_template.php already merged from bootstrap theme and latest pull for theme.php file for the same theme (that add font awesome 4 icons in this theme) and please update here this post.

**Comment by tgtje** [comment]
Ok, was just to inform what is happening ( it could help investigate). I will revert any changes and start over..

**Comment by tgtje** [comment]
Replaced and tested

standard with only the theme php (pull) ( and with or without forum css > no real change)

image 1 : 
![newa1](https://cloud.githubusercontent.com/assets/1811857/8396859/9a1a9634-1db8-11e5-92a1-b934b5746659.jpg)

So all is there now (apart from styling on visible posts)
At this point i still miss the admin area icon cog : used (pull) theme shortcodes) .. refreshed > it is there !!

Just for information as mentioned earlier : using the new bootsrat cdn 3.3.5  displays image : 2

![ane3](https://cloud.githubusercontent.com/assets/1811857/8396869/eda27b8c-1db8-11e5-8f42-0cc346ec3542.jpg)

Would need overthinking if e107 'upgrades' to new bootstrap min css (not up to me)

So basically if the pull requests are in..  working again.
Thanks for the investigation and work .

**Comment by mcpeace-maw** [comment]
I'll update bootstrap theme_shortcodes file for all 4 old icons.
Yes, I think the two bootstrap themes need new bootstrap version. Waiting for green flag to update the 2 themes.

**Comment by mcpeace-maw** [comment]
If theme_shortcodes pull is merged all icons are OK in bootstrap theme front end. In Admin Area I see just ADLAN_113 and ADLAN_115 in status menu that are commented in lan_admin.php file and don't work. They need to be uncommented in lan_admin.php to work. The rest seams to be OK. Will be nice if we can use font icons in this area too.

**Comment by willem010** [comment]
all the icon / font awesome issues, different versions of bootstrap etc.
this is why i opted for the autosprite thingie (a long time ago already)

this would render all these problems solved, and make e107 more optimized without dependancies.

just saying .. haha ;)

**Comment by mcpeace-maw** [comment]
For compatibility are use also images icons in non bootstrap themes, but today using image icons is not a good option. And for same compatibility are used 2 versions of bootstrap, and I think that is not good too. But these are just my opinions.

**Comment by Moc** [comment]
Bootstrap 2 is phased out and will be removed. It's just the admin theme that's still depending on it but that will be sorted out. As for fontawesome, just stick to the latest version.

**Comment by mcpeace-maw** [comment]
Well it is not exactly what I see, featurebox plugin need to stay compatible with old bootstrap, see my pulls for this plugin.

**Comment by Moc** [comment]
Yeah I'm just reading up on it. I was under the impression that e107 would go for bs3. Will look into this.

**Comment by mcpeace-maw** [comment]
I think that need to be closed, all is OK now, right?

**Comment by Moc** [comment]
@tgtje Is the issue resolved using the latest files?

**Comment by tgtje** [comment]
For me it is resolved (thanks @mcpeace-maw  for the works) , the use of BS vs BS3 (cdn) is debatable  (like the newer styling with cdn 3.3.5 more). So i will close this  (can be reopened if needed).
